src/Sylius/Behat/Client/ContentTypeGuide.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Client;
+
+use Symfony\Component\HttpFoundation\Request as HttpRequest;
+
+class ContentTypeGuide implements ContentTypeGuideInterface
```

Comment:
```suggestion
final class ContentTypeGuide implements ContentTypeGuideInterface
```

While having only one public method there is no benefit of having class not finalized